js/background.js

```diff
@@ -978,6 +978,30 @@
       }
     });
 
+    Whisper.events.on('onShowUserDetails', async ({ userPubKey }) => {
+      const conversation = await ConversationController.getOrCreateAndWait(
+        userPubKey,
+        'private'
+      );
+
+      const avatarPath = conversation.getAvatarPath();
+      const profile = conversation.getLokiProfile();
+      const displayName = profile && profile.displayName;
+
+      if (appView) {
+        appView.showUserDetailsDialog({
+          profileName: displayName,
+          pubkey: userPubKey,
+          avatarPath,
+          avatarColor: conversation.getColor(),
+          onOk: async () => {},
```

Comment:
I'm maybe missing something, but do we still need the `onOk`?
I guess it's been replaced with  `onStartConversation`?

Reply:
Absolutely!